### ISSUE TITLE
Parent domain: / not found

### ISSUE THREAD

**Issue by dorukozturk** [HDFGroup/hsds#13]
Hi,

I am trying the [docker installation instructions](https://github.com/HDFGroup/hsds/blob/master/docs/docker_install.rst).

When I run hsinfo, I get:
```
2019-01-01 12:00:48,172 set log_level to 10
2019-01-01 12:00:48,173 GET: http://hsds.hdf.test/about [None]
2019-01-01 12:00:48,178 Starting new HTTP connection (1): hsds.hdf.test:80
2019-01-01 12:00:48,185 http://hsds.hdf.test:80 "GET /about HTTP/1.1" 200 165
2019-01-01 12:00:48,186 status: 200
server name: hsdstest
server state: READY
endpoint: http://hsds.hdf.test
username: admin
password: *****
2019-01-01 12:00:48,187 GET: http://hsds.hdf.test/ [/home]
2019-01-01 12:00:48,189 Starting new HTTP connection (1): hsds.hdf.test:80
2019-01-01 12:00:48,292 http://hsds.hdf.test:80 "GET /?domain=%2Fhome HTTP/1.1" 404 14
2019-01-01 12:00:48,293 status: 404
2019-01-01 12:00:48,294 status_code: 404
Error: [Errno 404] Not Found
```

And following that when I do hstouch /home/, I get 
```
2019-01-01 12:02:16,324 set log_level to 10
2019-01-01 12:02:16,324 GET: http://hsds.hdf.test/domains [None]
2019-01-01 12:02:16,329 Starting new HTTP connection (1): hsds.hdf.test:80
2019-01-01 12:02:16,408 http://hsds.hdf.test:80 "GET /domains HTTP/1.1" 404 14
2019-01-01 12:02:16,411 status: 404
2019-01-01 12:02:16,411 status_code: 404
Parent domain: / not found
```

I feel like I am skipping some step to create the root domain.
Any help would be appreciated. 
Thanks.

**Comment by jreadey** [comment]
Did you follow step 17, ``hstouch /home/``?
After doing this, you should see the folder created in the S3 bucket.

**Comment by dorukozturk** [comment]
Yes, that is the step that I am getting stuck at. When I run that I see 
```
Parent domain: / not found
```

**Comment by jreadey** [comment]
Strange - try running this: `` python create_toplevel_domain_json.py --domain=/home``.  Note, no trailing slash on /home.

**Comment by jreadey** [comment]
Glad you are up and running!

**Comment by seb5g** [comment]
hello John,
I'm using the posix version of docker-compose and I got same error:
`(base) weber@weber-Precision-WorkStation-T3400:~$ hsinfo -e http://hsds.hdf.test
server name: hsdstest
server state: READY
endpoint: http://hsds.hdf.test
username: None
password: 
server version: 0.6_beta
node count: 1
up: 40 min 34 sec
h5pyd version: 0.7.1
(base) weber@weber-Precision-WorkStation-T3400:~$ hsinfo -e http://hsds.hdf.test -u test_user1 -p test
server name: hsdstest
server state: READY
endpoint: http://hsds.hdf.test
username: test_user1
password: ****
Error: [Errno 404] Not Found
(base) weber@weber-Precision-WorkStation-T3400:~$ 
`

I don't quite get where the error comes from. By the way with functions such as hsinfo or hstouch... I have to specify the end point (shoud it not see it as an env variable as defined in bashrc?) Or should I create a configuration file?

**Comment by seb5g** [comment]
oh I actually found the error: on the doc you write to export the variable 
16. Export the admin password as ADMIN_PASSWD

while the code expect ADMIN_PASSWORD ! With this the tests are all ok and hsinfo is fine also.

but the endpoint is still not known ...
Cheers

**Comment by jreadey** [comment]
@seb5g - sorry about that - I just updated the docs.

If after install you run hsconfigure you can enter values for endpoint, username, and password and it will save to the file .hscfg in your home directory.
